frontend/src/components/core/Block/Block.tsx

```diff
@@ -225,13 +220,12 @@ class Block extends PureComponent<Props> {
         widgetsDisabled={this.props.widgetsDisabled}
         componentRegistry={this.props.componentRegistry}
         formsData={this.props.formsData}
-        formsMgr={this.props.formsMgr}
         {...optionalProps}
       />
     )
 
-    if (isValidFormId(node.deltaBlock.formId)) {
-      const { formId } = node.deltaBlock
+    if (node.deltaBlock.type === "form") {
```

Comment:
should we keep the block types in an enum?

Reply:
They are defined as a type union, so they're type-safe: 
```
public type?: ("vertical"|"horizontal"|"column"|"expandable"|"form")
```